src/main/java/de/bonndan/nivio/input/ItemRelationProcessor.java

```diff
@@ -45,7 +49,7 @@ public ProcessingChangelog process(@NonNull final LandscapeDescription input, @N
                         })
                         .orElseGet(() -> {
                             Relation created = RelationFactory.create(origin, relationDescription, landscape);
-                            processLog.info(String.format(origin + ": Adding relation between %s and %s", created.getSource(), created.getTarget()));
+                            processLog.info(String.format("%s: Adding relation between %s and %s", origin, created.getSource(), created.getTarget()));
```

Comment:
@bonndan wenn hier Performance wichtig ist, würde ich die Strings konkatenieren, auch wenn dadurch die Lesbarkeit leidet.

Reply:
Good point, but here we don't use string concatenation very often. We'll keep it in mind when tackling #688.